Sources/SwiftDocC/Semantics/Snippets/Snippet.swift

```diff
@@ -18,18 +18,25 @@ public final class Snippet: Semantic, DirectiveConvertible {
         enum Path: DirectiveArgument {
             static let argumentName = "path"
         }
+        enum Slice: DirectiveArgument {
+            static let argumentName = "slice"
+        }
     }
 
     public let originalMarkup: BlockDirective
     public let path: String
+    public let slice: String?
```

Comment:
Not specific to this PR, but having documentation comments for these public APIs would be great.

Reply:
No problem, added.